flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/delegation/ParserImplTest.java

```diff
@@ -114,6 +115,18 @@ public void testCompletionTest() {
         verifySqlCompletion("SELECT a fram b", 10, new String[] {"FETCH", "FROM"});
     }
 
+    @Test
+    public void testSqlLike() {
```

Comment:
Add a separate test class `SqlLikeUtilsTest` for this test because it's not related to the parser.